common/src/main/java/io/homeassistant/companion/android/common/assist/AssistViewModelBase.kt

```diff
@@ -51,6 +52,7 @@ abstract class AssistViewModelBase(
 
     private var binaryHandlerId: Int? = null
     private var conversationId: String? = null
+    private var continueConversation = AtomicBoolean(false)
```

Comment:
I decided to use an AtomicBoolean here, because we are not 100% sure about the threading that is coming from the flow (it can change over time while developing) and the value is modified in two different places. To avoid potential issues I preferred to be safe.